xml/repeated-content-decl.ent

```diff
@@ -1195,6 +1195,131 @@ nginx-ingress-controller   LoadBalancer   <replaceable>10.63.248.70</replaceable
   </procedure>
  </sect2>'>
 
+<!--ENTITY affinity..........................................................-->
+
+<!ENTITY affinity
+'<title xmlns="http://docbook.org/ns/docbook">
+  Affinity and Anti-affinity
+ </title>
+ <para xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+  Operators can set affinity/anti-affinity rules to restrict how the scheduler
+  determines the placement of a given pod on a given node. This can be
+  achieved through node affinity/anti-affinity, where placement is determined
+  by node labels (see
+  <link xlink:href="https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity"/>),
+  or pod affinity/anti-affinity, where pod placement is determined by labels
+  on pods that are already running on the node (see
+  <link xlink:href="https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity"/>). 
+ </para>
+ <para xmlns="http://docbook.org/ns/docbook">
+  In &productname;, a default configuration of will have following
```

Comment:
I assume the `of` is extraneous here.

Reply:
Yup. Feel free to fix directly so you can merge